test/cartesian/YAxis.spec.tsx

```diff
@@ -40,12 +41,14 @@ describe('<YAxis />', () => {
     expect(ticks[1].getAttribute('y')).toBe('297.5');
   });
 
-  each([
+  const cases1: [number, AxisDomain, string][] = [
```

Comment:
It would be great to use a better name here.

Reply:
Done!